pkg/doc/verifiable/credential.go

```diff
@@ -187,12 +191,39 @@ type Credential struct {
 	RefreshService *RefreshService
 }
 
+// JWTAlgorithm defines JWT signature algorithms of Verifiable Credential
+type JWTAlgorithm int
+
+const (
+	// ES256K JWT Algorithm
+	ES256K JWTAlgorithm = iota
+	// RS256 JWT Algorithm
+	RS256
+	// EdDSA JWT Algorithm
+	EdDSA
+)
+
+// Jose converts JWTAlgorithm to JOSE one.
+func (ja JWTAlgorithm) Jose() jose.SignatureAlgorithm {
+	switch ja {
+	case ES256K:
+		return jose.ES256
```

Comment:
to be verified, I don't think `jose.ES256` is ES256k

Reply:
Yes, true! Filed #381